index.html

```diff
@@ -92,10 +92,61 @@ <h2 class="main-form-title">Заинтересовались?</h2>
+                <path fill-rule="evenodd" clip-rule="evenodd" d="M4.59399 9.22498H7.21899V11.291H4.59399V9.22498ZM4.59399 12.668H7.21899V14.735H4.59399V12.668ZM7.21899 16.112H4.59399V18.178H7.21899V16.112ZM9.18799 16.112H11.813V18.178H9.18799V16.112ZM11.813 12.668H9.18799V14.735H11.813V12.668ZM9.18799 9.22498H11.813V11.291H9.18799V9.22498ZM16.406 16.112H13.781V18.178H16.406V16.112ZM13.781 12.668H16.406V14.735H13.781V12.668ZM16.406 9.22498H13.781V11.291H16.406V9.22498Z" fill="#A9A9A9"/>
+              </svg>
+            </div>
+            <div class="form-item">
+              <label for="adult">Взрослые</label>
+              <div class="form-number">
+                <button class="form-number-item munus">
+                  <svg width="12" height="3" viewBox="0 0 12 3" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <rect width="12" height="3" fill="#A9A9A9"/>
+                  </svg>
+              </button>
+              <input type="text" id="adult" value="1">
+              <button class="form-number-item plus">
+                <svg width="11" height="11" viewBox="0 0 11 11" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd" clip-rule="evenodd" d="M4.23086 6.76932V11H6.76932V6.76932H11V4.23086H6.76932V0H4.23085V4.23086H0V6.76932H4.23086Z" fill="#A9A9A9"/>
+                </svg>
+            </button>
+          </div>
+          <label for="children">Дети</label>
+          <div class="form-number">
+            <button class="form-number-item munus">
+              <svg width="12" height="3" viewBox="0 0 12 3" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <rect width="12" height="3" fill="#A9A9A9"/>
+              </svg>
+            </button>
+            <input type="text" id="children" value="1">
+            <button class="form-number-item plus">
+              <svg width="11" height="11" viewBox="0 0 11 11" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <path fill-rule="evenodd" clip-rule="evenodd" d="M4.23086 6.76932V11H6.76932V6.76932H11V4.23086H6.76932V0H4.23085V4.23086H0V6.76932H4.23086Z" fill="#A9A9A9"/>
+              </svg>
+          </button>
+        </div>
+            </div>
+            <input type="submit" value="Найти" class="form-button">
```

Comment:
Это устаревший способ. Делай всегда кнопкой. 
```suggestion
            <button class="form-button">Найти</button>
```